test/e2e/network/dns_common.go

```diff
@@ -416,19 +424,24 @@ func createProbeCommand(namesToResolve []string, hostEntries []string, ptrLookup
 	for _, name := range hostEntries {
 		fileName := fmt.Sprintf("%s_hosts@%s", fileNamePrefix, name)
 		fileNames = append(fileNames, fileName)
-		probeCmd += fmt.Sprintf(`test -n "$$(getent hosts %s)" && echo OK > /results/%s;`, name, fileName)
+		probeCmd += fmt.Sprintf(`test -n "$$(getent %s %s)" && echo OK > /results/%s;`, getentdb, name, fileName)
+	}
+
+	podARecByUDPFileName := fmt.Sprintf("%s_udp@Pod%sRecord", fileNamePrefix, typeA)
+	podARecByTCPFileName := fmt.Sprintf("%s_tcp@Pod%sRecord", fileNamePrefix, typeA)
+	if isIpv6 {
+		probeCmd += fmt.Sprintf(`podARec="$$(ifconfig eth0  | awk '/inet6/{print $3}' | sed s/\:/-/g | sed -e's/\/.*$//g' | head -n 1 | awk '{print $1".%s.pod.cluster.local"}')";`, namespace)
```

Comment:
I'm guessing that the global IP address should be listed first, but would it be safer to grep on "[Gg]lobal" to select the right IP address ,instead?